File: C#/Lv_0/배열 자르기.cs
using System;

public class 배열_자르기 {
    public int[] solution(int[] numbers, int num1, int num2) {
        int k = num2-num1+1;
        int[] answer = new int[k];
        int j = 0;
        for(int i=num1; i<=num2; i++)
        {
            answer[j] = numbers[i];
            j++;
        }
        return answer;
    }
}

File: JavaScript/Lv_0/배열 자르기.js
function solution(numbers, num1, num2) {
    return numbers.splice(num1, num2-num1+1);
}

// 또는 slice를 이용한다.
// function solution(numbers, num1, num2) {
//     return numbers.slice(num1, num2+1);
// }